plugins/experimental/epic/epic.cc

```diff
@@ -50,120 +50,192 @@
+  "proxy.process.cluster.connections_avg_time",
+  "proxy.process.cluster.connections_open",
+  "proxy.process.cluster.control_messages_avg_receive_time",
+  "proxy.process.cluster.control_messages_avg_send_time",
+  "proxy.process.cluster.lkrmt_cache_callback_time",
+  "proxy.process.cluster.local_connection_time",
+  "proxy.process.cluster.open_delay_time",
+  "proxy.process.cluster.rdmsg_assemble_time",
+  "proxy.process.cluster.remote_connection_time",
+  "proxy.process.cluster.remote_op_reply_timeouts",
+  "proxy.process.cluster.remote_op_timeouts",
+  "proxy.process.cluster.rmt_cache_callback_time",
+  "proxy.process.dns.fail_avg_time",
+  "proxy.process.dns.in_flight",
+  "proxy.process.dns.lookup_avg_time",
+  "proxy.process.dns.success_avg_time",
+  "proxy.process.hostdb.total_entries",
+  "proxy.process.http.avg_transactions_per_client_connection",
+  "proxy.process.http.avg_transactions_per_parent_connection",
+  "proxy.process.http.avg_transactions_per_server_connection",
+  "proxy.process.http.background_fill_current_count",
+  "proxy.process.http.current_active_client_connections",
+  "proxy.process.http.current_cache_connections",
+  "proxy.process.http.current_client_connections",
+  "proxy.process.http.current_client_transactions",
+  "proxy.process.http.current_icp_raw_transactions",
+  "proxy.process.http.current_icp_transactions",
+  "proxy.process.http.current_parent_proxy_connections",
+  "proxy.process.http.current_parent_proxy_raw_transactions",
+  "proxy.process.http.current_parent_proxy_transactions",
+  "proxy.process.http.current_server_connections",
+  "proxy.process.http.current_server_raw_transactions",
+  "proxy.process.http.current_server_transactions",
+  "proxy.process.http.origin_server_speed_bytes_per_sec_100",
+  "proxy.process.http.origin_server_speed_bytes_per_sec_100K",
+  "proxy.process.http.origin_server_speed_bytes_per_sec_100M",
+  "proxy.process.http.origin_server_speed_bytes_per_sec_10K",
+  "proxy.process.http.origin_server_speed_bytes_per_sec_10M",
+  "proxy.process.http.origin_server_speed_bytes_per_sec_1K",
+  "proxy.process.http.origin_server_speed_bytes_per_sec_1M",
+  "proxy.process.http.user_agent_speed_bytes_per_sec_100",
+  "proxy.process.http.user_agent_speed_bytes_per_sec_100K",
+  "proxy.process.http.user_agent_speed_bytes_per_sec_100M",
+  "proxy.process.http.user_agent_speed_bytes_per_sec_10K",
+  "proxy.process.http.user_agent_speed_bytes_per_sec_10M",
+  "proxy.process.http.user_agent_speed_bytes_per_sec_1K",
+  "proxy.process.http.user_agent_speed_bytes_per_sec_1M",
+  "proxy.process.log.log_files_open",
+  "proxy.process.log.log_files_space_used",
+  "proxy.process.net.accepts_currently_open",
+  "proxy.process.net.connections_currently_open",
+  "proxy.process.socks.connections_currently_open",
+  "proxy.process.update.state_machines",
+  "proxy.process.version.server.build_time",
+  "proxy.process.websocket.current_active_client_connections",
+};
 
```

Comment:
> On Sep 13, 2016, at 5:34 PM, Susan Hinrichs <notifications@github.com> wrote:
> 
> In plugins/experimental/epic/epic.cc:
> 
> > +  "proxy.process.http.user_agent_speed_bytes_per_sec_100",
> > +  "proxy.process.http.user_agent_speed_bytes_per_sec_100K",
> > +  "proxy.process.http.user_agent_speed_bytes_per_sec_100M",
> > +  "proxy.process.http.user_agent_speed_bytes_per_sec_10K",
> > +  "proxy.process.http.user_agent_speed_bytes_per_sec_10M",
> > +  "proxy.process.http.user_agent_speed_bytes_per_sec_1K",
> > +  "proxy.process.http.user_agent_speed_bytes_per_sec_1M",
> > +  "proxy.process.log.log_files_open",
> > +  "proxy.process.log.log_files_space_used",
> > +  "proxy.process.net.accepts_currently_open",
> > +  "proxy.process.net.connections_currently_open",
> > +  "proxy.process.socks.connections_currently_open",
> > +  "proxy.process.update.state_machines",
> > +  "proxy.process.version.server.build_time",
> > +  "proxy.process.websocket.current_active_client_connections",
> > +};
> >  
> Isn't this kind of an unrelated change?
> 
Yeh I made it because the clang format made the original diff really big. I'll reorder the commits which should make the functional one much clearer.